javascript-build-logic/build.gradle.kts

```diff
@@ -3,15 +3,20 @@ plugins {
 }
 
 repositories {
+    google {
+        content {
+            includeGroupByRegex("com\\.android.*")
+            includeGroupByRegex("com\\.google.*")
+            includeGroupByRegex("androidx.*")
+        }
+    }
     mavenCentral()
-    google()
-
     gradlePluginPortal()
 }
 
 dependencies {
-    api("dev.icerock:mobile-multiplatform:0.14.1")
-    api("org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.21")
-    api("com.android.tools.build:gradle:7.0.1")
-    api("io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.15.0")
+    implementation(libs.mokoMultiplatform)
+    implementation(libs.kotlinGradlePlugin)
+    implementation(libs.androidGradlePlugin)
+    implementation(libs.detektPlugin)
```

Comment:
why implementation instead of api?

Reply:
i not found reasons for api, need rollback?